.github/labeler.yaml

```diff
@@ -1,16 +1,26 @@
 github-actions:
-- '.github/**'
-- '.goreleaser.yaml'
+  - changed-files:
+      - any-glob-to-any-file:
+        - '.github/**'
+        - '.goreleaser.yaml'
 
-dependencies: 
-- 'go.mod'
-- 'go.sum'
+dependencies:
+  - changed-files:
+      - any-glob-to-any-file:
+        - 'go.mod'
+        - 'go.sum'
 
 documentation:
-- 'README.md'
+  - changed-files:
+      - any-glob-to-any-file:
+        - 'README.md'
 
 legal:
-- 'LICENSE'
+  - changed-files:
+      - any-glob-to-any-file:
+        - 'LICENSE'
 
 tests:
-- '**/*_test.go'
+  - changed-files:
+      - any-glob-to-any-file:
+        - '**/*_test.go'
```

Comment:
If it's "any glob to any file" isn't this annotation then incorrect?

Reply:
According to the docs it works as follows: any-glob-to-any-file: ANY glob must match against ANY changed file

Reply:
I'm just going ahead and approve this. It's super unclear, even after reading the docs myself I still find it overly vague.

Reply:
💯